docs/src/main/sphinx/release/release-479.md

```diff
@@ -0,0 +1,121 @@
+# Release 479 (TBD 2025)
+
+## General
+
+* Support automatic TLS certificate generation in the `ANNOUNCE` node discovery mode. ({issue}`27030`)
+* Add support for specifying the location of the Exchange Manager configuration file. ({issue}`26611`)
+* Publish query finishing time in event listener. ({issue}`27202`)
+* Add lineage support for output columns in all SELECT queries({issue}`26241`)
```

Comment:
Original release note in pr was: "Compute lineage for output columns in all SELECT queries." 
@kasiafi @Praveen2112 @behrooz-stripe  FYI